cpp/src/sampling/detail/shuffle_and_organize_output_impl.cuh

```diff
@@ -16,9 +16,11 @@
 
 #pragma once
 
+#include "cugraph/utilities/shuffle_comm.cuh"
 #include "prims/update_edge_src_dst_property.cuh"  // ??
```

Comment:
This might be from a previous PR but why ?? If we are not using this, can't we delete?